ee/clickhouse/models/test/__snapshots__/test_property.ambr

```diff
@@ -57,7 +57,7 @@
 ---
 # name: test_parse_groups_persons_edge_case_with_single_filter
   <class 'tuple'> (
-    'AND (  has(%(vglobalperson_0)s, "pmat_email"))',
+    'AND (  has(%(vglobalperson_0)s, replaceRegexpAll(JSONExtractRaw(person_props, %(kglobalperson_0)s), \'^"|"$\', \'\')))',
```

Comment:
That's an unexpected change 🤔

Reply:
[not happening just on this PR, but a few others as well, so ignoring here 🤔 ]